pkg/nacler/keyers/secureenclave/secureenclave.go

```diff
@@ -0,0 +1,194 @@
+//go:build darwin
+// +build darwin
+
+package secureenclave
+
+// this was heavily inspiried by https://github.com/facebookincubator/sks
+// thank you!
+
+/*
+#cgo darwin LDFLAGS: -framework Foundation -framework Security -framework CoreFoundation
+
+#include <stdlib.h>
+#include <secureenclave.h>
+
+typedef struct wrapper {
+	unsigned char *buf;
+	int status;
+	size_t size;
+	char *error;
+} Wrapper;
```

Comment:
What is the Wrapper for?

Reply:
I left a comment in the secureenclave.h file with some explanation.